File: 387-first-unique-character-in-a-string/387-first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
        
        int[] chars = new int[26];
        
        for(int i=0;i < s.length(); i++)
        {
            chars[s.charAt(i) - 'a']++;
        }
        
        for(int j=0; j < s.length(); j++)
        {
            if( chars[s.charAt(j) - 'a'] == 1)
                return j;            
        }
        
        return -1;
        
        
    }
}

File: 387-first-unique-character-in-a-string/387-first-unique-character-in-a-string.cs
public class Solution {
    public int FirstUniqChar(string s) {
        int[] chars = new int[26];
        
        for(int i=0;i < s.Length; i++)
        {            
            chars[s[i] - 'a']++;            
        }
        
        for(int j=0;j < s.Length; j++)
        {
            if(chars[s[j] - 'a']  == 1)
                return j;
        }
        
        return -1;
    }
}